### ISSUE TITLE
Problem building on AWS linux 2

### ISSUE THREAD

**Issue by Charles92011** [cisco/libsrtp#527]
After install openssl11 from the AWS repo, libsrtp configure gives the error:
```
configure: error: Package requirements (libcrypto >= 1.0.1) were not met:
No package 'libcrypto' found
```

A search for libcrypto reveals these results:
```
/usr/lib64/.libcrypto.so.1.0.2k.hmac
/usr/lib64/.libcrypto.so.1.1.1g.hmac
/usr/lib64/.libcrypto.so.1.1.hmac
/usr/lib64/.libcrypto.so.10.hmac
/usr/lib64/libcrypto.so
/usr/lib64/libcrypto.so.1.0.2k
/usr/lib64/libcrypto.so.1.1
/usr/lib64/libcrypto.so.1.1.1g
/usr/lib64/libcrypto.so.10
/usr/lib64/pkgconfig/libcrypto11.pc
```

**Comment by paulej** [comment]
I just fired up a 64-bit image selecting the Amazon Linux image offered at the top of the launch page.  I installed git, gcc and openssl-devel.  I then ran configure and make.  All worked fine.  However, what got installed was openssl 1.02k.  How did you install openssl 1.1?  Amazon's Linux did not give me that option.

I have OpenSSL 1.1.1i installed on a Fedora instance and it builds just fine there.  So whatever the issue is, definitely not something related to OpenSSL 1.1, per se.
